### DISCUSSION TITLE
How to get the Default Console Width?

### DISCUSSION THREAD

**Discussion by Guillermo-Santos** [CosmosOS/Cosmos#2733] [Q&A] [answered]
I am writing a method to write text, but if the line length is greater than the console Width then it needs to be splitetd (not by character, but by word). but when i try to get them from the console using any of the next lines, the System crash:
``` Csharp
Console.Write($"BufferWidth {Console.BufferWidth}");
Console.Write($"WindowWidth {Console.WindowWidth}");
```
<img width="677" alt="image" src="https://github.com/CosmosOS/Cosmos/assets/96314559/aed4bf3a-9782-4f1a-943b-f2df72c02d4c">
<img width="337" alt="image" src="https://github.com/CosmosOS/Cosmos/assets/96314559/a8f2c9d8-f70b-4992-af37-50a49f9f52a4">

Is there a workaround for this?

**Comment by HyperWinX** [comment, marked as answer]
Try Console.WindowWidth and Console.WindowHeight

**Reply by Guillermo-Santos** [reply]
Jajaja turns out it was using the old compilation that used Console.BufferWidth even though i was making change to the code, strange...

But now it works!
<img width="638" alt="image" src="https://github.com/CosmosOS/Cosmos/assets/96314559/41b6d33b-f7a0-4a5d-a7df-af3e96222dbd">

<img width="662" alt="image" src="https://github.com/CosmosOS/Cosmos/assets/96314559/31494f53-ee44-485f-bfc7-fb793243ba01">

Still curious as to why  Console.BufferWidth failed.
